java-conductr-bundle-lib/src/main/java/com/typesafe/conductr/bundlelib/java/Env.java

```diff
@@ -0,0 +1,26 @@
+package com.typesafe.conductr.bundlelib.java;
```

Comment:
Can't we use the `com.typesafe.conductr.bundlelib.Env` directly if we change the interface towards returning `Optional<T>`?. We then only need to modify the Scala interface to return `Option<T>`.